src/test/java/seedu/address/model/module/TutorialClassTest.java

```diff
@@ -187,4 +187,39 @@ void deleteStudent_inList_success() {
         assertTrue(tutorialClass.deleteStudent(student1));
         assertEquals(0, tutorialClass.getStudents().size());
     }
+
+    @Test
+    void deleteStudentFromTeams_success() {
+        ArrayList<Person> students = new ArrayList<>();
+
+        // Creating Person object
+        Name name1 = new Name("John");
+        Email email1 = new Email("john@example.com");
+        StudentId stuId1 = new StudentId("A0213333J");
+        Set<Tag> tags1 = new HashSet<>();
+        Person student1 = new Person(name1, email1, stuId1, tags1);
+
+        students.add(student1);
+
+        TutorialClass tutorialClass = new TutorialClass(VALID_TUTORIAL, students);
+
+        // Create teams, add student to teams, and add teams to tutorial class
+        TutorialTeam team1 = new TutorialTeam("a");
+        TutorialTeam team2 = new TutorialTeam("b");
```

Comment:
there are valid team names in the command util class to use, you may want to use that instead!

Reply:
Alright, will update